homeassistant/components/cloud/helpers.py

```diff
@@ -0,0 +1,31 @@
+"""Helpers for the cloud component."""
+
+from collections import deque
+import logging
+
+from homeassistant.core import HomeAssistant
+
+
+class FixedSizeQueueLogHandler(logging.Handler):
```

Comment:
What makes this cloud specific?
Would this not be useful for others like mqtt?

Reply:
If someone wants to implement this for more integrations we can move this out of the cloud integration. But I don't we usually add stuff to the top level helpers if they are not used by multiple integrations.

Reply:
If it becomes useful for others, we can extract it and make it generic. In general, we don't make things generic if they are used only for a single purpose.

Reply:
The entire support pacakge thing that are being added for cloud, I want in all integrations Im codeowner for, I assume most code owners would like much of the same level of data.

Reply:
For that purpose, something else was already partly built. This feature is specifically scoped for Cloud for now.